src/algebra/continued_fractions/computation/approximation_corollaries.lean

```diff
@@ -0,0 +1,144 @@
+/-
+Copyright (c) 2021 Kevin Kappelmann. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Kevin Kappelmann
+-/
+import algebra.continued_fractions.computation.approximations
+import algebra.continued_fractions.convergents_equiv
+import topology.instances.ennreal
+/-!
+# Corollaries Following From Approximation Lemmas in `approximations.lean`
+
+## Summary
+
+Let us write `gcf` for `generalized_continued_fraction`.
+We show that the generalized_continued_fraction given by `gcf.of` in fact is a (regular) continued
+fraction. Using the equivalence of the convergents computations (`gcf.convergents` and
+`gcf.convergents'`) for continued fractions (see `algebra.continued_fractions.convergents_equiv`),
+it follows that the convergents computations for `gcf.of` are equivalent.
+
+Moreover, we show the convergence of the continued fractions computations, that is
+`(gcf.of v).convergents` indeed computes `v` in the limit.
+
+## Main Definitions
+
+- `continued_fraction.of` returns the (regular) continued fraction of a value.
+
+## Main Theorems
+
+- `gcf.of_convergents_eq_convergents'` shows that the convergents computations for `gcf.of` are
```

Comment:
Please expand the `gcf` abbreviation here. This items are automatically turned into links by the documentation tool, but for that to work, they need to mention the actual Lean name of the result.

Reply:
Fixed it in all files of `algebra/continued_fractions/`